testing/test_aux.sh

```diff
@@ -57,8 +57,8 @@ fail_hook=misc/dump_network.sh
 test_config=misc/runtime_configs/long_wait
 site_path=inst/test_site
 startup_faux_1_opts="brute"
-startup_faux_2_opts="nobrute expiredtls bacnetfail pubber"
-startup_faux_3_opts="tls macoui bacnet pubber"
+startup_faux_2_opts="nobrute expiredtls bacnetfail pubber passwordpass"
+startup_faux_3_opts="tls macoui passwordfail bacnet pubber"
```

Comment:
swap these around 
generally 
faux_1 is all skip
faux_2 is all fail
fuax_3 is all pass

Reply:
fixed